Gems/PhysX/Code/Tests/ShapeColliderComponentTests.cpp

```diff
@@ -151,6 +152,245 @@ namespace PhysXEditorTests
         EXPECT_TRUE(aabb.GetMin().IsClose(-0.5f * boxDimensions));
     }
 
+    TEST_F(PhysXEditorFixture, EditorShapeColliderComponent_ShapeColliderWithBoxAndTranslationOffset_CorrectEditorStaticBodyGeometry)
+    {
+        AZ::Transform transform(AZ::Vector3(2.0f, -6.0f, 5.0f), AZ::Quaternion(0.3f, -0.3f, 0.1f, 0.9f), 1.6f);
+        AZ::Vector3 nonUniformScale(2.0f, 2.5f, 0.5f);
+        AZ::Vector3 boxDimensions(5.0f, 8.0f, 6.0f);
+        AZ::Vector3 translationOffset(-4.0f, 3.0f, -1.0f);
+        EntityPtr editorEntity = CreateBoxShapeColliderEditorEntity(transform, nonUniformScale, boxDimensions, translationOffset);
+
+        AzPhysics::SimulatedBody* simulatedBody = nullptr;
+        AzPhysics::SimulatedBodyComponentRequestsBus::EventResult(
+            simulatedBody, editorEntity->GetId(), &AzPhysics::SimulatedBodyComponentRequests::GetSimulatedBody);
+        const auto* pxRigidStatic = static_cast<const physx::PxRigidStatic*>(simulatedBody->GetNativePointer());
+
+        AZ::Aabb aabb = PxMathConvert(pxRigidStatic->getWorldBounds(1.0f));
+        EXPECT_THAT(aabb.GetMin(), UnitTest::IsClose(AZ::Vector3(-25.488f, -10.16f, -11.448f)));
+        EXPECT_THAT(aabb.GetMax(), UnitTest::IsClose(AZ::Vector3(1.136f, 18.32f, 16.584f)));
```

Comment:
question: Just wanted to check where these numbers came from, did you run this in a debugger first or verify in the editor or just calculate by hand?

Reply:
All figured out by hand.

Reply:
Lol I should have known 😅 Sorry I doubted you 😉